x-pack/plugins/reporting/server/routes/main.js

```diff
@@ -6,7 +6,7 @@
 
 import boom from 'boom';
 import { API_BASE_URL } from '../../common/constants';
-import { enqueueJobFactory } from '../lib/enqueue_job';
+import { enqueueJobFactory } from '../lib/perch_enqueue_job';
```

Comment:
mebbe fence as well?
```suggestion
/* BEGIN PERCH CODE */
import { enqueueJobFactory } from '../lib/perch_enqueue_job';
/* END PERCH CODE */
```